pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>se.lfv.reqstool</groupId>
+	<artifactId>reqstool-demo</artifactId>
+	<version>0.0.1</version>
```

Comment:
We should bump the version so that it is unique (can't recall if we used 0.0.1 internally or not).

Reply:
yes